ros/src/util/packages/runtime_manager/scripts/computing.yaml

```diff
@@ -2303,6 +2315,36 @@ params :
         delim        : ':='
         only_enable  : True
 
+  - name : region_tlr_tensorflow
+    topic: /config/superimpose
+    msg  : Bool
+    vars :
+    - name  : data
+      desc  : data desc sample
+      label : Display superimpose result
+      kind  : checkbox
+      v     : False
+    - name         : camera_id
+      desc         : camera_id desc sample
+      kind         : menu
+      label        : Camera ID
+      choices_type : str
+      v            : ''
+      cmd_param    :
+        dash         : ''
+        delim        : ':='
+        only_enable  : True
+    - name      : nn_model_path
+      desc      : network model file
+      label     : nn_model_path
+      kind      : path
+      path_type : multi
+      v         : /home/dev/config/model-RGB-009-0.9857.hdf5
+      cmd_param :
+        dash        : ''
+        delim       : ':='
+
+
```

Comment:
This currently won't work for 2 reasons:

1. This node no longer listens to the `/config/superimpose` topic. All it did was pop up an OpenCV window so we removed it.
2. These `/config` topics are theoretically for things that can be dynamically reconfigured. The NN model is not one of those things right now. We didn't design the node to handle changing the model on-the-fly - just at start-up. Same goes for the camera_id at the moment - though that would be easier to change.

Reply:
The NN model path is a static option and it is not going to be part of the /config/superimpose topic (I can get rid of that). We can leave the model path on there to provide the option to set a path to the model from the GUI. We can include other launch file parameters in here once we have the final version, even if not to dynamically reconfigure right now. Not all nodes in autoware have a config message yet. What do you think @JWhitleyAStuff ?

Reply:
That doesn't make sense. We have seen that changes to these GUI fields publish a configuration message, which we don't support. There is no point in adding these here since they only change that configuration message. Is there documentation on what these fields mean? I was to understand that these fields were *only* for dynamic reconfiguration. Do you have some information that says different?